docs/kyma-environment-broker/03-01-service-description.md

```diff
@@ -111,23 +92,30 @@ These are the provisioning parameters for GCP that you can configure:
 | **autoScalerMax** | int | Specifies the maximum number of virtual machines to create. | No | `4` |
 | **maxSurge** | int | Specifies the maximum number of virtual machines that are created during an update. | No | `4` |
 | **maxUnavailable** | int | Specifies the maximum number of VMs that can be unavailable during an update. | No | `1` |
+ 
+ </details>
+ </div>
+
      
-  </details>
+## Trial Plan
+
+Trial Plan allows the user to choose the provider where they want to install Kyma. Trial drawbacks are that Kyma will be
+uninstalled, and the cluster will be deprovisioned after 30 days. Apart from that it's possible to provision only one Kyma
+per Global Account.
+
+These are the provisioning parameters for Trial Plan that you can configure:
+  
+<div tabs name="trial-plan" group="trial-plan">
   <details>
-  <summary label="gcp-trial-plan">
-  GCP Trial
+  <summary label="trial-plan">
+  Trial Plan
   </summary>
     
-| Parameter Name | Type  | Description |  Required  | Default value |
+| Parameter Name | Type | Description | Required | Possible values| Default value |
 | ---------------|-------|-------------|:----------:|---------------|
-| **machineType**    | string | Specifies the provider-specific virtual machine type. | No | `n1-standard-4` |
-| **volumeSizeGb** | int | Specifies the size of the root volume. | No | `30` |
-| **region**         | string | Defines the cluster region. | No | `europe-west4` |
-| **zones**          | string | Defines the list of zones in which the Runtime Provisioner creates the cluster. | No | `["a", "b", "c"]` |
-| **autoScalerMin**  | int    | Specifies the minimum number of virtual machines to create. | No | `3` |
-| **autoScalerMax**  | int    | Specifies the maximum number of virtual machines to create. | No | `3` |
-| **maxSurge**       | int    | Specifies the maximum number of virtual machines that are created during an update. | No | `1` |
-| **maxUnavailable** | int    | Specifies the maximum number of VMs that can be unavailable during an update. | No | `1` |
-     
-  </details>
-</div>
+| **name** | string | Specifies the provider-specific virtual machine type. | No | Any string| `n1-standard-4` |
+| **region** | int | Specifies the size of the root volume. | No | `europe`,`us` | `30` |
+| **provider** | string | Defines the cluster region. | No | `Azure`, `GCP` | `europe-west4` |
```

Comment:
adjust description, and I think you missed some columns, grid is not rendering correctly